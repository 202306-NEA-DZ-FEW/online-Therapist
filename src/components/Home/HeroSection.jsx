import Image from "next/image";
import Slider from "react-slick";

import "slick-carousel/slick/slick.css";
import "slick-carousel/slick/slick-theme.css";
import { slideImages } from "@/util/library";
import Button from "../elements/Button";
import Link from "next/link";
import { useTranslation } from "next-i18next";
import { UserAuth } from "@/context/AuthContext";
import { useRouter } from "next/router";

function HeroSection() {
    const router = useRouter();
    const { user } = UserAuth();

    const handleBooking = () => {
        // Check if the user is logged in
        if (!user) {
            router.push("/login/");
            // window.alert("You need to be logged in to Book an appointment");
        } else {
            router.push("/bookAnAppointment/");
        }
    };
    // Define the paths based on the user's authentication status
    // const bookingPath = user ? "/bookAnAppointment" : "/login/login";
    const { t } = useTranslation("homepage");
    const settings = {
        fade: false,
        infinite: true,
        arrows: false,
        autoplay: true,
        autoplaySpeed: 5000,
        pauseOnHover: false,
        slidesToShow: 1,
        slidesToScroll: 1,
        initialSlide: 0,
    };

    return (
        <div>
        <Slider {...settings} className='bg-white'>
            {slideImages.map((img) => (
                <div key={img.id}>
                    <div
                        style={{
                            backgroundImage: `url(/Images/${img.imgSrc}.jpg)`,
                        }}
                        className='h-[90vh] bg-cover bg-center relative'
                    ></div>
                </div>
            ))}
        </Slider>
        <div className='flex flex-col justify-between absolute leading-loose bottom-[25%] left-8 right-8 text-center lg:left-48 lg:right-48 p-6 bg-opacity-50 bg-white bg-clip-padding'>
            <div className='space-y-4'>
                <h2 className='text-4xl md:text-4xl lg:text-7xl  text-black font-extrabold font-atkinson'>
                    {t("hero.heading")}
                    <br />
                </h2>
                <p className='hidden md:block md:text-xl lg:text-3xl sm:text-md font-medium text-black leading-loose font-atkinson'>
                    {t("hero.text1")}
                </p>
                <p className='hidden md:block text-lg md:text-xl lg:text-3xl font-medium  font-atkinson leading-loose'>
                    {t("hero.text2")}
                </p>
            </div>
            <div className='px-auto pt-6 mx-auto'>
                <Button
                    color='teal'
                    transition={true}
                    buttonText={t("hero.bookingButton")}
                    buttonSize='fit'
                    clickFunction={handleBooking}
                />
            </div>
        </div>
    </div>
    );
}

export default HeroSection;
