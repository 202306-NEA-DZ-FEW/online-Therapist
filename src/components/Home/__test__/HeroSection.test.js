import renderer from "react-test-renderer";
import "./matchMedia.mock";

import HeroSection from "../HeroSection";

it("renders correctly", () => {
    const tree = renderer.create(<HeroSection />).toJSON();
    expect(tree).toMatchSnapshot();
});
