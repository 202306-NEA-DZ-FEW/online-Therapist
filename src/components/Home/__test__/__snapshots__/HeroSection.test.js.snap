// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="slick-slider bg-GreenLight slick-initialized"
  dir="ltr"
>
  <div
    className="slick-list"
    onClick={[Function]}
    onKeyDown={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={null}
    onMouseMove={null}
    onMouseUp={[Function]}
    onTouchCancel={null}
    onTouchEnd={[Function]}
    onTouchMove={null}
    onTouchStart={[Function]}
    style={Object {}}
  >
    <div
      className="slick-track"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseOver={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <div
        aria-hidden={false}
        className="slick-slide slick-active slick-current"
        data-index={0}
        onClick={[Function]}
        style={
          Object {
            "left": -0,
            "opacity": 1,
            "outline": "none",
            "position": "relative",
            "transition": "opacity 500ms ease, visibility 500ms ease",
            "width": 0,
          }
        }
        tabIndex="-1"
      >
        <div>
          <div
            style={
              Object {
                "display": "inline-block",
                "width": "100%",
              }
            }
            tabIndex={-1}
          >
            <img
              alt="Therapist Sessions"
              className="object-cover w-full max-h-[80vh]"
              data-nimg="1"
              decoding="async"
              height={400}
              loading="lazy"
              onError={[Function]}
              onLoad={[Function]}
              src="/_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FuBaE00%2Fza%2F3475714.jpg&w=828&q=75"
              srcSet="/_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FuBaE00%2Fza%2F3475714.jpg&w=640&q=75 1x, /_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FuBaE00%2Fza%2F3475714.jpg&w=828&q=75 2x"
              style={
                Object {
                  "color": "transparent",
                }
              }
              width={400}
            />
            <div
              className=" flex justify-between absolute bottom-1.5 text-left left-0 right-0 p-4 bg-opacity-50 bg-white bg-clip-padding"
            >
              <div
                className=""
              >
                <h2
                  className="text-lg md:text-xl lg:text-3xl sm:text-md text-DarkTeal font-extrabold font-atkinson"
                >
                  Therapy at Your Fingertips: Access the World's Largest Network of Therapists
                </h2>
                <p
                  className="hidden sm:block text-lg md:text-xl lg:text-2xl font-opensans"
                >
                  Get the same quality care as in-office therapy, but with the convenience and flexibility of online.
                </p>
              </div>
              <div
                className="px-auto pt-4 mx-auto"
              >
                <div
                  className="relative inline-flex items-center justify-center px-6 py-3 overflow-hidden font-medium transition duration-300 ease-out border-2 border-Teal rounded-lg shadow-md group text-xl w-32 h-14 bg-white hover:cursor-pointer"
                >
                  <span
                    className="absolute inset-0 flex items-center justify-center w-full h-full text-white duration-300 -translate-x-full bg-Teal group-hover:translate-x-0 ease"
                  >
                    <svg
                      className="w-6 h-6"
                      fill="none"
                      stroke="currentColor"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M14 5l7 7m0 0l-7 7m7-7H3"
                        strokeLinecap="round"
                        strokeLinejoin="round"
                        strokeWidth="2"
                      />
                    </svg>
                  </span>
                  <span
                    className="absolute flex items-center justify-center w-full h-full text-Teal transition-all duration-300 transform group-hover:translate-x-full ease"
                  />
                  <span
                    className="relative invisible"
                  >
                    $
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        aria-hidden={true}
        className="slick-slide"
        data-index={1}
        onClick={[Function]}
        style={
          Object {
            "left": -0,
            "opacity": 0,
            "outline": "none",
            "position": "relative",
            "transition": "opacity 500ms ease, visibility 500ms ease",
            "width": 0,
          }
        }
        tabIndex="-1"
      >
        <div>
          <div
            style={
              Object {
                "display": "inline-block",
                "width": "100%",
              }
            }
            tabIndex={-1}
          >
            <img
              alt="Therapist Sessions"
              className="object-cover w-full max-h-[80vh]"
              data-nimg="1"
              decoding="async"
              height={400}
              loading="lazy"
              onError={[Function]}
              onLoad={[Function]}
              src="/_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FHVAM00%2Fza%2F5283595.jpg&w=828&q=75"
              srcSet="/_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FHVAM00%2Fza%2F5283595.jpg&w=640&q=75 1x, /_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FHVAM00%2Fza%2F5283595.jpg&w=828&q=75 2x"
              style={
                Object {
                  "color": "transparent",
                }
              }
              width={400}
            />
            <div
              className=" flex justify-between absolute bottom-1.5 text-left left-0 right-0 p-4 bg-opacity-50 bg-white bg-clip-padding"
            >
              <div
                className=""
              >
                <h2
                  className="text-lg md:text-xl lg:text-3xl sm:text-md text-DarkTeal font-extrabold font-atkinson"
                >
                  Therapy at Your Fingertips: Access the World's Largest Network of Therapists
                </h2>
                <p
                  className="hidden sm:block text-lg md:text-xl lg:text-2xl font-opensans"
                >
                  Get the same quality care as in-office therapy, but with the convenience and flexibility of online.
                </p>
              </div>
              <div
                className="px-auto pt-4 mx-auto"
              >
                <div
                  className="relative inline-flex items-center justify-center px-6 py-3 overflow-hidden font-medium transition duration-300 ease-out border-2 border-Teal rounded-lg shadow-md group text-xl w-32 h-14 bg-white hover:cursor-pointer"
                >
                  <span
                    className="absolute inset-0 flex items-center justify-center w-full h-full text-white duration-300 -translate-x-full bg-Teal group-hover:translate-x-0 ease"
                  >
                    <svg
                      className="w-6 h-6"
                      fill="none"
                      stroke="currentColor"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M14 5l7 7m0 0l-7 7m7-7H3"
                        strokeLinecap="round"
                        strokeLinejoin="round"
                        strokeWidth="2"
                      />
                    </svg>
                  </span>
                  <span
                    className="absolute flex items-center justify-center w-full h-full text-Teal transition-all duration-300 transform group-hover:translate-x-full ease"
                  />
                  <span
                    className="relative invisible"
                  >
                    $
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        aria-hidden={true}
        className="slick-slide"
        data-index={2}
        onClick={[Function]}
        style={
          Object {
            "left": -0,
            "opacity": 0,
            "outline": "none",
            "position": "relative",
            "transition": "opacity 500ms ease, visibility 500ms ease",
            "width": 0,
          }
        }
        tabIndex="-1"
      >
        <div>
          <div
            style={
              Object {
                "display": "inline-block",
                "width": "100%",
              }
            }
            tabIndex={-1}
          >
            <img
              alt="Therapist Sessions"
              className="object-cover w-full max-h-[80vh]"
              data-nimg="1"
              decoding="async"
              height={400}
              loading="lazy"
              onError={[Function]}
              onLoad={[Function]}
              src="/_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FGEqF00%2Fza%2F3775692.jpg&w=828&q=75"
              srcSet="/_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FGEqF00%2Fza%2F3775692.jpg&w=640&q=75 1x, /_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FGEqF00%2Fza%2F3775692.jpg&w=828&q=75 2x"
              style={
                Object {
                  "color": "transparent",
                }
              }
              width={400}
            />
            <div
              className=" flex justify-between absolute bottom-1.5 text-left left-0 right-0 p-4 bg-opacity-50 bg-white bg-clip-padding"
            >
              <div
                className=""
              >
                <h2
                  className="text-lg md:text-xl lg:text-3xl sm:text-md text-DarkTeal font-extrabold font-atkinson"
                >
                  Therapy at Your Fingertips: Access the World's Largest Network of Therapists
                </h2>
                <p
                  className="hidden sm:block text-lg md:text-xl lg:text-2xl font-opensans"
                >
                  Get the same quality care as in-office therapy, but with the convenience and flexibility of online.
                </p>
              </div>
              <div
                className="px-auto pt-4 mx-auto"
              >
                <div
                  className="relative inline-flex items-center justify-center px-6 py-3 overflow-hidden font-medium transition duration-300 ease-out border-2 border-Teal rounded-lg shadow-md group text-xl w-32 h-14 bg-white hover:cursor-pointer"
                >
                  <span
                    className="absolute inset-0 flex items-center justify-center w-full h-full text-white duration-300 -translate-x-full bg-Teal group-hover:translate-x-0 ease"
                  >
                    <svg
                      className="w-6 h-6"
                      fill="none"
                      stroke="currentColor"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M14 5l7 7m0 0l-7 7m7-7H3"
                        strokeLinecap="round"
                        strokeLinejoin="round"
                        strokeWidth="2"
                      />
                    </svg>
                  </span>
                  <span
                    className="absolute flex items-center justify-center w-full h-full text-Teal transition-all duration-300 transform group-hover:translate-x-full ease"
                  />
                  <span
                    className="relative invisible"
                  >
                    $
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        aria-hidden={true}
        className="slick-slide"
        data-index={3}
        onClick={[Function]}
        style={
          Object {
            "left": -0,
            "opacity": 0,
            "outline": "none",
            "position": "relative",
            "transition": "opacity 500ms ease, visibility 500ms ease",
            "width": 0,
          }
        }
        tabIndex="-1"
      >
        <div>
          <div
            style={
              Object {
                "display": "inline-block",
                "width": "100%",
              }
            }
            tabIndex={-1}
          >
            <img
              alt="Therapist Sessions"
              className="object-cover w-full max-h-[80vh]"
              data-nimg="1"
              decoding="async"
              height={400}
              loading="lazy"
              onError={[Function]}
              onLoad={[Function]}
              src="/_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FVSOH00%2Fza%2F4145599.jpg&w=828&q=75"
              srcSet="/_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FVSOH00%2Fza%2F4145599.jpg&w=640&q=75 1x, /_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FVSOH00%2Fza%2F4145599.jpg&w=828&q=75 2x"
              style={
                Object {
                  "color": "transparent",
                }
              }
              width={400}
            />
            <div
              className=" flex justify-between absolute bottom-1.5 text-left left-0 right-0 p-4 bg-opacity-50 bg-white bg-clip-padding"
            >
              <div
                className=""
              >
                <h2
                  className="text-lg md:text-xl lg:text-3xl sm:text-md text-DarkTeal font-extrabold font-atkinson"
                >
                  Therapy at Your Fingertips: Access the World's Largest Network of Therapists
                </h2>
                <p
                  className="hidden sm:block text-lg md:text-xl lg:text-2xl font-opensans"
                >
                  Get the same quality care as in-office therapy, but with the convenience and flexibility of online.
                </p>
              </div>
              <div
                className="px-auto pt-4 mx-auto"
              >
                <div
                  className="relative inline-flex items-center justify-center px-6 py-3 overflow-hidden font-medium transition duration-300 ease-out border-2 border-Teal rounded-lg shadow-md group text-xl w-32 h-14 bg-white hover:cursor-pointer"
                >
                  <span
                    className="absolute inset-0 flex items-center justify-center w-full h-full text-white duration-300 -translate-x-full bg-Teal group-hover:translate-x-0 ease"
                  >
                    <svg
                      className="w-6 h-6"
                      fill="none"
                      stroke="currentColor"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M14 5l7 7m0 0l-7 7m7-7H3"
                        strokeLinecap="round"
                        strokeLinejoin="round"
                        strokeWidth="2"
                      />
                    </svg>
                  </span>
                  <span
                    className="absolute flex items-center justify-center w-full h-full text-Teal transition-all duration-300 transform group-hover:translate-x-full ease"
                  />
                  <span
                    className="relative invisible"
                  >
                    $
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        aria-hidden={true}
        className="slick-slide"
        data-index={4}
        onClick={[Function]}
        style={
          Object {
            "left": -0,
            "opacity": 0,
            "outline": "none",
            "position": "relative",
            "transition": "opacity 500ms ease, visibility 500ms ease",
            "width": 0,
          }
        }
        tabIndex="-1"
      >
        <div>
          <div
            style={
              Object {
                "display": "inline-block",
                "width": "100%",
              }
            }
            tabIndex={-1}
          >
            <img
              alt="Therapist Sessions"
              className="object-cover w-full max-h-[80vh]"
              data-nimg="1"
              decoding="async"
              height={400}
              loading="lazy"
              onError={[Function]}
              onLoad={[Function]}
              src="/_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FuBaE00%2Fza%2F3475714.jpg&w=828&q=75"
              srcSet="/_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FuBaE00%2Fza%2F3475714.jpg&w=640&q=75 1x, /_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FuBaE00%2Fza%2F3475714.jpg&w=828&q=75 2x"
              style={
                Object {
                  "color": "transparent",
                }
              }
              width={400}
            />
            <div
              className=" flex justify-between absolute bottom-1.5 text-left left-0 right-0 p-4 bg-opacity-50 bg-white bg-clip-padding"
            >
              <div
                className=""
              >
                <h2
                  className="text-lg md:text-xl lg:text-3xl sm:text-md text-DarkTeal font-extrabold font-atkinson"
                >
                  Therapy at Your Fingertips: Access the World's Largest Network of Therapists
                </h2>
                <p
                  className="hidden sm:block text-lg md:text-xl lg:text-2xl font-opensans"
                >
                  Get the same quality care as in-office therapy, but with the convenience and flexibility of online.
                </p>
              </div>
              <div
                className="px-auto pt-4 mx-auto"
              >
                <div
                  className="relative inline-flex items-center justify-center px-6 py-3 overflow-hidden font-medium transition duration-300 ease-out border-2 border-Teal rounded-lg shadow-md group text-xl w-32 h-14 bg-white hover:cursor-pointer"
                >
                  <span
                    className="absolute inset-0 flex items-center justify-center w-full h-full text-white duration-300 -translate-x-full bg-Teal group-hover:translate-x-0 ease"
                  >
                    <svg
                      className="w-6 h-6"
                      fill="none"
                      stroke="currentColor"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M14 5l7 7m0 0l-7 7m7-7H3"
                        strokeLinecap="round"
                        strokeLinejoin="round"
                        strokeWidth="2"
                      />
                    </svg>
                  </span>
                  <span
                    className="absolute flex items-center justify-center w-full h-full text-Teal transition-all duration-300 transform group-hover:translate-x-full ease"
                  />
                  <span
                    className="relative invisible"
                  >
                    $
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        aria-hidden={true}
        className="slick-slide"
        data-index={5}
        onClick={[Function]}
        style={
          Object {
            "left": -0,
            "opacity": 0,
            "outline": "none",
            "position": "relative",
            "transition": "opacity 500ms ease, visibility 500ms ease",
            "width": 0,
          }
        }
        tabIndex="-1"
      >
        <div>
          <div
            style={
              Object {
                "display": "inline-block",
                "width": "100%",
              }
            }
            tabIndex={-1}
          >
            <img
              alt="Therapist Sessions"
              className="object-cover w-full max-h-[80vh]"
              data-nimg="1"
              decoding="async"
              height={400}
              loading="lazy"
              onError={[Function]}
              onLoad={[Function]}
              src="/_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FEUzL00%2Fza%2F5241246.jpg&w=828&q=75"
              srcSet="/_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FEUzL00%2Fza%2F5241246.jpg&w=640&q=75 1x, /_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FEUzL00%2Fza%2F5241246.jpg&w=828&q=75 2x"
              style={
                Object {
                  "color": "transparent",
                }
              }
              width={400}
            />
            <div
              className=" flex justify-between absolute bottom-1.5 text-left left-0 right-0 p-4 bg-opacity-50 bg-white bg-clip-padding"
            >
              <div
                className=""
              >
                <h2
                  className="text-lg md:text-xl lg:text-3xl sm:text-md text-DarkTeal font-extrabold font-atkinson"
                >
                  Therapy at Your Fingertips: Access the World's Largest Network of Therapists
                </h2>
                <p
                  className="hidden sm:block text-lg md:text-xl lg:text-2xl font-opensans"
                >
                  Get the same quality care as in-office therapy, but with the convenience and flexibility of online.
                </p>
              </div>
              <div
                className="px-auto pt-4 mx-auto"
              >
                <div
                  className="relative inline-flex items-center justify-center px-6 py-3 overflow-hidden font-medium transition duration-300 ease-out border-2 border-Teal rounded-lg shadow-md group text-xl w-32 h-14 bg-white hover:cursor-pointer"
                >
                  <span
                    className="absolute inset-0 flex items-center justify-center w-full h-full text-white duration-300 -translate-x-full bg-Teal group-hover:translate-x-0 ease"
                  >
                    <svg
                      className="w-6 h-6"
                      fill="none"
                      stroke="currentColor"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M14 5l7 7m0 0l-7 7m7-7H3"
                        strokeLinecap="round"
                        strokeLinejoin="round"
                        strokeWidth="2"
                      />
                    </svg>
                  </span>
                  <span
                    className="absolute flex items-center justify-center w-full h-full text-Teal transition-all duration-300 transform group-hover:translate-x-full ease"
                  />
                  <span
                    className="relative invisible"
                  >
                    $
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        aria-hidden={true}
        className="slick-slide"
        data-index={6}
        onClick={[Function]}
        style={
          Object {
            "left": -0,
            "opacity": 0,
            "outline": "none",
            "position": "relative",
            "transition": "opacity 500ms ease, visibility 500ms ease",
            "width": 0,
          }
        }
        tabIndex="-1"
      >
        <div>
          <div
            style={
              Object {
                "display": "inline-block",
                "width": "100%",
              }
            }
            tabIndex={-1}
          >
            <img
              alt="Therapist Sessions"
              className="object-cover w-full max-h-[80vh]"
              data-nimg="1"
              decoding="async"
              height={400}
              loading="lazy"
              onError={[Function]}
              onLoad={[Function]}
              src="/_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FcBaE00%2Fza%2F3475696.jpg&w=828&q=75"
              srcSet="/_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FcBaE00%2Fza%2F3475696.jpg&w=640&q=75 1x, /_next/image?url=https%3A%2F%2Fc.stocksy.com%2Fa%2FcBaE00%2Fza%2F3475696.jpg&w=828&q=75 2x"
              style={
                Object {
                  "color": "transparent",
                }
              }
              width={400}
            />
            <div
              className=" flex justify-between absolute bottom-1.5 text-left left-0 right-0 p-4 bg-opacity-50 bg-white bg-clip-padding"
            >
              <div
                className=""
              >
                <h2
                  className="text-lg md:text-xl lg:text-3xl sm:text-md text-DarkTeal font-extrabold font-atkinson"
                >
                  Therapy at Your Fingertips: Access the World's Largest Network of Therapists
                </h2>
                <p
                  className="hidden sm:block text-lg md:text-xl lg:text-2xl font-opensans"
                >
                  Get the same quality care as in-office therapy, but with the convenience and flexibility of online.
                </p>
              </div>
              <div
                className="px-auto pt-4 mx-auto"
              >
                <div
                  className="relative inline-flex items-center justify-center px-6 py-3 overflow-hidden font-medium transition duration-300 ease-out border-2 border-Teal rounded-lg shadow-md group text-xl w-32 h-14 bg-white hover:cursor-pointer"
                >
                  <span
                    className="absolute inset-0 flex items-center justify-center w-full h-full text-white duration-300 -translate-x-full bg-Teal group-hover:translate-x-0 ease"
                  >
                    <svg
                      className="w-6 h-6"
                      fill="none"
                      stroke="currentColor"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M14 5l7 7m0 0l-7 7m7-7H3"
                        strokeLinecap="round"
                        strokeLinejoin="round"
                        strokeWidth="2"
                      />
                    </svg>
                  </span>
                  <span
                    className="absolute flex items-center justify-center w-full h-full text-Teal transition-all duration-300 transform group-hover:translate-x-full ease"
                  />
                  <span
                    className="relative invisible"
                  >
                    $
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
